testing/simapp/simd/cmd/genaccounts_test.go

```diff
@@ -69,7 +69,7 @@ func TestAddGenesisAccountCmd(t *testing.T) {
 			ctx = context.WithValue(ctx, client.ClientContextKey, &clientCtx)
 			ctx = context.WithValue(ctx, server.ServerContextKey, serverCtx)
 
-			cmd := simcmd.AddGenesisAccountCmd(home)
+			cmd := genutilcli.AddGenesisAccountCmd(home)
```

Comment:
Since we are now basically testing Cosmos SDK functionality that [they are also testing in their code](https://github.com/cosmos/cosmos-sdk/blob/main/x/genutil/client/cli/genaccount_test.go), would it make sense to just get rid of this file completely?

Reply:
Yes it makes sense